index.html

```diff
@@ -28,6 +28,142 @@ <h1>Iheagwara<br>Augustine</h1>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
+                </div>
+            </div>
+            <div class="project-items-4">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
+                </div>
+            </div>
+            <div class="project-items-5">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
+                </div>
+            </div>
+            <div class="project-items-6">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
+                    <button class="see-this-project">
+                        See this project
+                        <img src="images.svg\Union (7).svg" alt="" class="arrow">
+                    </button>
+                </div>
+            </div>
+        </section>
+        <section class="section-3">
+            <h1 class="about-sign">About</h1>
+            <p>
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ab culpa iste nam qui excepturi facere at quis porro odio nulla in quod dignissimos, sint cumque quo! Repellat ipsum nulla a.
+            </p>
+            <button class="Start-collab">
+                Get my CV
+                <img src="images.svg\ic_download (2).svg" alt="">
+            </button>
```

Comment:
- [ ] Please add a type of button to this button 
- [ ] the space between the image and the get my resume test is not the same as the one on the left, please fix that too.

Reply:
please explain type of button @joskalenda

Reply:
<img width="730" alt="1" src="https://user-images.githubusercontent.com/77843050/212503511-b0be399a-f002-4bb8-aed3-a1f36acb0b0c.png">



In your case, just do `<button type = button></button>` and it will be enough, if you need to read and have a small idea of what it is, feel free to follow this [link](https://html.com/attributes/button-type/)